### PR TITLE
* Remove features implemented for #1148

### PR DESCRIPTION
**Pull request by Wagnerp**
* Remove features implemented for #1148

image

### CODE REVIEW

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualTaskDialog.cs [hunk 1]
```diff
@@ -1227,7 +1227,7 @@ private void InitializeComponent()
             // 
             this._messageContent.AutoSize = false;
             this._messageContent.Font = new System.Drawing.Font("Segoe UI", 9F);
-            this._messageContent.ForeColor = System.Drawing.Color.White;
+            this._messageContent.ForeColor = System.Drawing.Color.FromArgb(((int)(((byte)(30)))), ((int)(((byte)(57)))), ((int)(((byte)(91)))));
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualTaskDialog.cs:1230]: The colour should be taken from the theme

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualTaskDialog.cs [hunk 2]
```diff
@@ -1432,7 +1432,7 @@ private void InitializeComponent()
             // 
             this._footerLabel.AutoSize = false;
             this._footerLabel.Font = new System.Drawing.Font("Segoe UI", 9F);
-            this._footerLabel.ForeColor = System.Drawing.Color.White;
+            this._footerLabel.ForeColor = System.Drawing.Color.FromArgb(((int)(((byte)(30)))), ((int)(((byte)(57)))), ((int)(((byte)(91)))));
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualTaskDialog.cs:1435]: Theme Colours....

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualTaskDialog.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualTaskDialog.cs:1230]: It does, but the designer code updates when the theme is changed. Should the `ForeColor` be serialised, i.e. hidden?

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualTaskDialog.cs (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualTaskDialog.cs:1435]: It does, but the designer code updates when the theme is changed. Should the `ForeColor` be serialised, i.e. hidden?

Comment (Smurf-IV) [reply Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualTaskDialog.cs:1435]: Try running these things on a Windows Light them and see what happens.
Default colours in the designer need to be form the the selected theme.
Forcing these overrides is confusing, thats why the other controls have "Init from theme" calls
If they are not to be set separately then they should be not be visible in the designer serialisation. as these things should just be controls (Like the rest of them in the KMessageBox)

Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualTaskDialog.cs:1435]: @Smurf-IV I'm going to recreate the `VisualTaskDialog`, as at the moment everything (designer & logic code) is 'shoved' into one huge class.

**Pull request merged**